obqa/commands/evaluate_predictions_qa_mc.py

```diff
@@ -231,7 +236,8 @@ def evaluate_from_args(args: argparse.Namespace) -> Dict[str, Any]:
         logger.info("Reading evaluation data from %s", evaluation_data_path)
         dataset = dataset_reader.read(evaluation_data_path)
 
-        metrics = evaluate(model, dataset, iterator, out_file, eval_type)
+        ## pass again the cuda device settings 
```

Comment:
Since I am anyways adding unimportant nitpicks, this seems to be a comment for the PR not the code. You can drop it (or change it to `# Evaluating model`)

Reply:
sure! I just removed the documentation and added a space after the comma. Let me know if there is anything else.